steward/src/cellars/cellar_v2_2.rs

```diff
@@ -176,6 +172,18 @@ pub fn get_encoded_function(call: FunctionCall, cellar_id: String) -> Result<Vec
 
             Ok(CellarV2_2Calls::InitiateShutdown(call).encode())
         }
+        Function::SetStrategistPlatformCut(params) => {
+            log_cellar_call(
+                CELLAR_NAME,
+                &SetStrategistPlatformCutCall::function_name(),
+                &cellar_id,
+            );
+            let call = SetStrategistPlatformCutCall {
```

Comment:
Strategist cut and platform fee both need specific values and specific cellar addresses.